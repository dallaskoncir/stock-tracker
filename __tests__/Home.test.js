import { render, unmountComponentAtNode } from 'react-dom';
import Home from '../pages/index';

let container = null;

beforeEach(() => {
    // setup a DOM element as a render target
    container = document.createElement('div');
    document.body.appendChild(container);
});

afterEach(() => {
    // cleanup on exiting
    unmountComponentAtNode(container);
    container.remove();
    container = null;
});

test('<Home /> should render correctly', () => {
    render(<Home />, container);

    expect(container).toMatchSnapshot();
});
