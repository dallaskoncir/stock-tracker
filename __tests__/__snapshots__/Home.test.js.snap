// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render correctly 1`] = `
<div>
  <main>
    <h1>
      Stock Comparison
    </h1>
    <p>
      Search and select up to 3 stocks to compare.
    </p>
    <div>
      <input
        type="text"
        value=""
      />
    </div>
  </main>
</div>
`;
